docs/source/daml/code-snippets/Reference.daml

```diff
@@ -1,12 +1,6 @@
 -- Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 -- SPDX-License-Identifier: Apache-2.0
 
--- template-let warning disabled because 'start template let snippet' is
--- used by docs.daml.com for older compiler versions and to explain
--- new deprecation warning in 2.8.0
```

Comment:
this was based on my misunderstanding of the `docs.daml.com` repo; in reality it uses specific commits of this repo so we don't need to keep this